tests/models/test_horovod.py

```diff
@@ -88,6 +88,7 @@ def test_horovod_cpu_implicit(tmpdir):
     _run_horovod(trainer_options)
 
 
+@pytest.mark.skipif(True, reason="fix hv")
```

Comment:
just `@pytest.mark.skip(reason="fix hv")`

Reply:
@Borda I removed it again in #2514 :)